operations/mimir-mixin/dashboards/dashboard-utils.libsonnet

```diff
@@ -816,7 +816,7 @@ local utils = import 'mixin-utils/utils.libsonnet';
         |||
           sum by(operation) (
             rate(
-              thanos_memcached_operations_total{
+              thanos_cache_operations_total{
```

Comment:
We need backward compatibility for all the queries in the dashboard. See my comment in the alert. You can use a similar strategy here too.